e2e/dotcms-e2e-node/frontend/utils/contentUtils.ts

```diff
@@ -306,6 +334,21 @@ interface FileAssetFormParams {
     newFileText?: string;
 }
 
+/**
+ * Parameters for the fillPageAssetForm method
+ */
+interface PageAssetFormParams {
+    page: Page;
+    title: string;
+    url?: string;
+    host?: string;
+    action?: string;
+    template?: string;
+    friendlyName?: string;
+    showOnMenu?: boolean;
+    sortOrder?: string;
+    cacheTTL?: number
+}
```

Comment:
I see that this and the interface above share some properties, maybe those properties can be shared using [Interface Extension](https://www.typescripttutorial.net/typescript-tutorial/typescript-extend-interface/) that way if you need to create another interface that uses the same properties, it will be easier to work with